### PR TITLE
LAB C108

### PR DESCRIPTION
**Pull request by Kgaur624**
Project convert into MVC based architecture

### CODE REVIEW

File: projecty/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
-Simple Logging Facade for Java (abbreviated SLF4J) – 
-acts as a facade for different logging frameworks
-It offers a generic API making the logging independent of the actual implementation.
-This allows for different logging frameworks to coexist. 
-It also helps migrate from one framework to another.
+The Model-View-Controller (MVC)
```
Comment (PoojaMohite-Khoros) [inline projecty/README.md:1]: Y this file changed ?

File: projecty/src/main/java/com/service/FetchTweets.java [hunk 2]
```diff
@@ -13,23 +11,27 @@
...
+     * parameter constructor are using for unit testing
+     * @param configurationBuilder
+     * @param twitterFactory
+     */
     public FetchTweets(ConfigurationBuilder configurationBuilder, TwitterFactory twitterFactory) {
         this.configurationBuilder = configurationBuilder;
         this.twitterFactory = twitterFactory;
     }
-
     public  Response latestTweet() {
```
Comment (PoojaMohite-Khoros) [inline projecty/src/main/java/com/service/FetchTweets.java:31]: Response return type should be only from controller class

File: projecty/src/main/java/com/service/Service.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    }
+
+    public TwitterFactory twitterFactory() {
+        return twitterFactory;
+    }
+
+    public Status status() throws TwitterException {
+        return twitter.updateStatus(postReq.getMessage());
+    }
+    public ResponseList<Status> timline() throws TwitterException {
```
Comment (PoojaMohite-Khoros) [inline projecty/src/main/java/com/service/Service.java:33]: ResponseList - instead use list and use response from controller

**Review by PoojaMohite-Khoros — CHANGES_REQUESTED** [review]

File: projecty/README.md (hunk 1 above)
Comment (Kgaur624) [reply projecty/README.md:1]: I have just added the description whatever I did in my project

**Review by anasuya1117 — APPROVED** [review]

**Pull request merged**